scripts/vm/thornsec.vm.control.stopVM

```diff
@@ -17,17 +17,26 @@ stopVm() {
             (( pullPlug-- ));
 	    sleep 1;
         done;
+	if sudo -u vboxuser_"${vm}" VBoxManage showvminfo "${vm}" --machinereadable | grep running

```

Comment:
Use ` vmRunning "${vm}" ` here as per L25 - if vmRunning doesn't do what you need, gimme a pull request :)

Merry Christmas, btw.